tasks/codegen/test_generate.py

```
@@ -0,0 +1,226 @@
+    assert "SET search_path" not in sql
+
+
+def test_ore_functions_file_counts_and_extractor(tmp_path):
+    spec = load(tmp_path)
+    ordered = next(d for d in spec.domains if d.name == "int4_ord")
+    sql = render_functions_file(spec, ordered)
+    assert sql.count("CREATE FUNCTION") == 31
+    assert "CREATE FUNCTION eql_v2.ord_term(a eql_v2_int4_ord)" in sql
+    assert "RETURNS eql_v2.ore_block_u64_8_256" in sql
+    # 1 extractor + 18 wrappers (=, <>, <, <=, >, >= across 3 shapes);
+    # 12 blockers (@>, <@ × 3 + path ops × 3) as plpgsql.
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)